src/foam/dao/LRUCachingDAO.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.dao',
+  name: 'LRUCachingDAO',
+  extends: 'foam.dao.ProxyDAO',
+  implements: [ 'foam.mlang.Expressions' ],
+
+  requires: [
+    'foam.dao.MDAO',
+    'foam.dao.QuickSink'
+  ],
+
+  documentation: `
+     Least Recently Used CachingDAO :
+     Manages a DAO\'s size by removing old items. Commonly applied inside a cache to limit the cache\'s size.
+  `,
+
+  classes: [
+    {
+      /** Links an object id to a last-accessed seqNo */
```

Comment:
can we use documentation rather than code comment? There are also a few similar places below.

Reply:
Yes, documentation: is better.